tandemhunter-main/generate_comparison_csv_to_xls.py

```diff
@@ -0,0 +1,88 @@
+"""
+script to generate .xls file from a run of cvg_comparison.csv
+generated from TandemHunter
+
+"""
+
+import argparse
+from pathlib import Path
+import pty
+import subprocess
+import os
+import sys
+import pytest
+
+import pandas as pd
+
+
+def parse_args():
+    """Parse command line arguments"""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--comparison_csv', type=argparse.FileType('r'), nargs="+",
+        required=True,
+        help='all cvg_comparison.csv files for tandemhunter output'
+    )
+    parser.add_argument(
+        '--run',
+        help='name of run to prefix output .xls file with'
```

Comment:
I can't find 'run' being used to prefix any xls file names inside the generate_comparison_csv_to_xls script. args.run doesn't appear in function KM2TA_df_to_xls, or in the earlier-called function generate_PTD_KMT2A_df. It may not have been written in to naming: if so, it could either be removed, or that prefixing function could be added.